docs/uninstall.md

```diff
@@ -0,0 +1,21 @@
+# Uninstalling AzureAuth
+
+We provide an uninstall script you can use to Uninstall AzureAuth. The script removes the default AzureAuth reference in the PATH and it will deletes the AzureAuth installation folder.
+
+The script currently doesn't support uninstalling from custom locations. This is to avoid removing any potential files that are not safe to delete. If there are any potential installations in custom locations found in the PATH (installed through the `$AZUREAUTH_INSTALL_DIRECTORY` environment variable), a warning will be prompted showing its location. Installations in custom locations that are not listed in the PATH cannot be found and uninstalled.
+
+## Usage
+
+Run the following in PowerShell:
+
+```PowerShell
+$script = "${env:TEMP}\uninstall.ps1"
+$url = "https://raw.githubusercontent.com/AzureAD/microsoft-authentication-cli/main/install/uninstall.ps1"
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Invoke-WebRequest $url -OutFile $script; if ($?) { &$script }; if ($?) { rm $script }
+```
+
+## Note
+
+* The script doesn't work for versions previous to AzureAuth 0.4.0
+* The script won't work as intended if you try to uninstall right after installing because PATH changes from the installation won't take effect until after restarting PowerShell.
```

Comment:
Is this true? If we uninstall immediately after installing I think we've touched the registry twice and just haven't updated the current terminal/console window (or potentially other processes). What's the actual end result of running uninstall immediately after install? 🤔

Reply:
I just tested this, and it does work. I though we encountered an issue with this in the bug bashing session, when Manuha had to reset her terminal. Maybe I am remembering wrong? Since both things work, I think these Note section. Thank you for pointing this out.

Reply:
Thank you @mvanchaa! I will remove the Note section then.